nni/algorithms/compression/v2/pytorch/pruning/movement_pruner.py

```diff
@@ -38,56 +38,12 @@ class PrunerScoredModuleWrapper(Module):
         The configurations that users specify for compression.
     module_name
         The name of the module to compress, wrapper module shares same name.
-    pruner
-        The pruner used to calculate mask.
     """
-    def __init__(self, module: Module, module_name: str, config: Dict, pruner: Compressor):
-        super().__init__()
-        # origin layer information
-        self.module = module
-        self.name = module_name
-        # config and pruner
-        self.config = config
-        self.pruner = pruner
-
-        self.weight = Parameter(torch.empty(self.module.weight.size()))
+    def __init__(self, module: Module, module_name: str, config: Dict):
+        super().__init__(module, module_name, config)
         self.weight_score = Parameter(torch.empty(self.weight.size()))
         torch.nn.init.constant_(self.weight_score, val=0.0)
 
-        # register buffer for mask
-        self.register_buffer("weight_mask", torch.ones(self.module.weight.shape))
-        if hasattr(self.module, 'bias') and self.module.bias is not None:
-            self.register_buffer("bias_mask", torch.ones(self.module.bias.shape))
-            self.bias = Parameter(torch.empty(self.module.bias.size()))
-        else:
-            self.register_buffer("bias_mask", None)
```

Comment:
so where are masks are generated then?

Reply:
all these logic are moved to its parent's class `PrunerModuleWrapper`

Reply:
got it